src/components/space/space.tsx

```diff
@@ -27,12 +27,17 @@ export const Space = () => {
       </Suspense>
       {satellites.map((satellite) => (
         <Fragment key={satellite.id}>
-          <Satellite color={satellite.color} tle={satellite.tle} />
+          <Satellite 
+            visibility={satellite.visibility}
+            path={satellite.path}
+            color={satellite.color}
+            tle={satellite.tle} />
           <Trajectory
             beginningDate={getTime() - 4000000}
             color={satellite.color}
             endDate={getTime() + 4000000}
             tle={satellite.tle}
+            visible={satellite.path}
```

Comment:
For consistency purposes `visibility` / `visible` prop name could be standardized.

> **Note**: React Three Fiber components have a `visible` prop.